src/Tools/Source/RunTests/TestHistoryManager.cs

```diff
@@ -23,15 +23,10 @@ internal class TestHistoryManager
     /// </summary>
     private const int MaxTestsReturnedPerRequest = 10_000;
 
-    /// <summary>
-    /// The pipeline id for roslyn-ci, see https://dev.azure.com/dnceng/public/_build?definitionId=15
-    /// </summary>
-    private const int RoslynCiBuildDefinitionId = 15;
-
     /// <summary>
     /// The Azure devops project that the build pipeline is located in.
     /// </summary>
-    private static readonly Uri s_projectUri = new(@"https://dev.azure.com/dnceng");
+    private static readonly Uri s_projectUri = new(@"https://dev.azure.com/dnceng-public");
```

Comment:
Could we pull this from the `System.CollectionUri` [predefined variable](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#system-variables-devops-services)?

Reply:
It probably should - as soon as I can figure out why its still not working now

Reply:
Ah I know why - its because we didn't have a passing build for main yet!

Reply:
updated and tested locally.  this should work once we get a passing main CI build (see https://github.com/dotnet/roslyn/pull/63851)